### PR TITLE
Add Quality Assurance Roadmap

### PR DESCRIPTION
**Pull request by fityanos**
[Testing] Adding Software Testing & QA Roadmap repository link

**[Insert URL to the list here]**
https://github.com/fityanos/awesome-quality-assurance-roadmap

**[Explain what this list is about and why it should be included here]**
This repository includes:
- Rich roadmap highlights software testing and software QA.
- Test Plan sample pdf for downloading and private usage.
- Translations to support community.

### CODE REVIEW

**Comment by fityanos** [comment]
unicorn

**Comment by fityanos** [comment]
reviewed PRs:
[#1912](https://github.com/sindresorhus/awesome/pull/1912#issuecomment-745865199)
[#1794](https://github.com/sindresorhus/awesome/pull/1794#issuecomment-745874379)

**Comment by sindresorhus** [comment]
Thanks for making an Awesome list! 🙌

It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

**Comment by sindresorhus** [comment]
> reviewed PRs:

Please try harder. You didn't even look at that PR...

**Comment by fityanos** [comment]
> > reviewed PRs:
> 
> Please try harder. You didn't even look at that PR...

Apologies for this, I posted a comment regarding missing unicorn but I removed it as it's not related

**Comment by fityanos** [comment]
> Thanks for making an Awesome list! 🙌
> 
> It looks like you didn't read the [guidelines](https://github.com/sindresorhus/awesome/blob/master/pull_request_template.md) closely enough. I noticed multiple things that are not followed. Try going through the list point for point to ensure you follow it. I spent a lot of time creating the guidelines so I wouldn't have to comment on common mistakes, and rather spend my time improving Awesome.

Thanks, @sindresorhus 

All is good now and ready for merge (I think) except the MIT as it cannot be changed as far as I know

**Comment by vmax** [comment]
@fityanos you should update this PR to include the new link ("https://github.com/fityanos/awesome-quality-assurance-road-map"), not the old one ("https://github.com/fityanos/Quality-Assurance-Road-Map#readme")

**Comment by fityanos** [comment]
> @fityanos you should update this PR to include the new link ("https://github.com/fityanos/awesome-quality-assurance-road-map"), not the old one ("https://github.com/fityanos/Quality-Assurance-Road-Map#readme")

Updated, thanks @vmax

File: readme.md [hunk 1]
```diff
@@ -599,6 +599,7 @@
 - [JMeter](https://github.com/aliesbelik/awesome-jmeter#readme) - Load testing and performance measurement tool.
 - [k6](https://github.com/k6io/awesome-k6#readme) - Open-source, developer-centric performance monitoring and load testing solution.
 - [Playwright](https://github.com/mxschmitt/awesome-playwright#readme) - Node.js library to automate Chromium, Firefox and WebKit with a single API.
+- [Software Testing & QA Roadmap](https://github.com/fityanos/Quality-Assurance-Road-Map#readme) - How to start & build a career in software testing.
```
Comment (sindresorhus) [inline readme.md:602]: The title here does not match the readme header.

**Comment by sindresorhus** [comment]
The list readme needs a better introduction.

**Comment by sindresorhus** [comment]
> 200~OK is not always okay

This means nothing without more context.

> Understand software testing and do not jump into automation

Same here. Completely meanless without an explanation.

**Comment by sindresorhus** [comment]
Road Map => Roadmap

**Comment by sindresorhus** [comment]
The PDF is fine, but more content needs to be in the actual readme.

File: readme.md (hunk 1 above)
Comment (fityanos) [reply readme.md]: fixed along with URL update.

**Comment by fityanos** [comment]
@sindresorhus Thanks for your feedback, all addressed

**Comment by sindresorhus** [comment]
> Road Map => Roadmap

This is not fixed everywhere. This includes the repo name.

**Comment by sindresorhus** [comment]
The PR title is incorrect.

**Comment by fityanos** [comment]
> > Road Map => Roadmap
> 
> This is not fixed everywhere. This includes the repo name.

Fixed

**Comment by fityanos** [comment]
> The PR title is incorrect.

Fixed

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1369567007998320644

**Pull request merged**